README.md

```diff
@@ -97,12 +96,12 @@ Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsp
 - [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later)
 
 **Docker (Linux)**: 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)
+
 **Docker (Windows)**: 1.13/17.03
 
-## Reference
+## References
 
 * **Known Issues**: Please check [vDVS known issue page](http://vmware.github.io/docker-volume-vsphere/documentation/known-issues.html) to find out about known issues.
 
-* **Contact us**: Please [click here](http://vmware.github.io/docker-volume-vsphere/documentation/contactus.html) for requesting any feature or reporting a product issue.
-
- * **Blogs**: Please check our [vDVS blog page](http://vmware.github.io/docker-volume-vsphere/documentation/blogs.html).
```

Comment:
IMO, it is better to have reference of "Blogs" in README.md

Reply:
List of blogs will get outdated really fast and gives an impression that we approve everything said in those blogs so I prefer not to keep it.

Reply:
Yeah. Blogs were added in absence of documentation. Now that we have docs, let's remove blogs.